api/src/bigquerytest/java/org/pmiops/workbench/api/DataSetControllerBQTest.java

```diff
@@ -232,11 +235,9 @@ public void setUp() {
         .thenReturn(fcResponse)
         .thenReturn(fcResponse);
 
-    dbCdrVersion = new DbCdrVersion();
+    dbCdrVersion = TestMockFactory.createDefaultCdrVersion(cdrVersionDao, accessTierDao);
```

Comment:
`createDefaultCdrVersion` associates the CDR Version with a Registered Access Tier